cloudflare/resource_cloudflare_teams_accounts_test.go

```diff
@@ -0,0 +1,70 @@
+package cloudflare
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccCloudflareTeamsAccountConfigurationBasic(t *testing.T) {
```

Comment:
```suggestion
func TestAccCloudflareTeamsAccountBasic(t *testing.T) {
```

Reply:
this test also seems to be failing the integration tests

```
TF_ACC=1 go test $(go list ./...) -v -run "^(TestAccCloudflareTeamsRuleBasic|TestAccCloudflareTeamsAccountConfigurationBasic)" -count 1 -timeout 120m -parallel 1
?   	github.com/cloudflare/terraform-provider-cloudflare	[no test files]
=== RUN   TestAccCloudflareTeamsAccountConfigurationBasic
    testing.go:705: Step 0 error: Check failed: Check 3/9 error: cloudflare_teams_account.iaxdohpgah: Attribute 'activity_log_enabled' expected "true", got "false"
--- FAIL: TestAccCloudflareTeamsAccountConfigurationBasic (1.27s)
=== RUN   TestAccCloudflareTeamsRuleBasic
--- PASS: TestAccCloudflareTeamsRuleBasic (2.98s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/cloudflare	4.604s
?   	github.com/cloudflare/terraform-provider-cloudflare/version	[no test files]
FAIL
make: *** [testacc] Error 1
```

Reply:
ahh that field is controlled by a flipper that is enabled on my account. I removed it from the test.